CPS-XXXX/README.md

```diff
@@ -0,0 +1,155 @@
+---
+CPS: ????
+Title: CPS-???? | Cardano URIs
+Status: Open
+Category: Tools
+Authors:
+  - Adam Dean <adam@crypto2099.io>
+Proposed Solutions:
+  - CIP-0013: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0013
+  - CIP-0099: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0099
+  - CIP-0107: https://github.com/cardano-foundation/CIPs/pull/635
```

Comment:
This looks fine, although for the record no CPS includes both the CIP name _and_ link in the list.  YAML parsing itself doesn't complain about it, so nothing seems to get in the way of interpretation as a "list" as defined by CIP-9999 (probably meaning "list" in the HTML element sense).

Only CPS-0005 and CPS-0007 have listed solutions, already in 2 different forms: and this makes a third.  But again I believe that should be no problem since neither of the popular derived CIP sites (Dev Portal & `cips.cardano.org`) are processing this keyword in CPS headers.

Reply:
Adding labels is quite a nice quality of life improvement for any links in any headers, if this doesnt break the yml we should look at this for all header links

Reply:
I wasn't sure what it was going to look like but very satisfied with how it turned out

Reply:
I also wouldn't have guessed it would be rendered like that & generally also like the `label:` convention for bypassing the prohibition against using Markdown in the YAML header.  For instance, instead of (illegal):
```
Implementors:
  - [Mlabs](https://mlabs.city)
```
we could have (legally):
```
Implementors:
  - Mlabs: https://mlabs.city
```
... not SEO as an anchored link would be, but the next best thing: a clickable link next to a clear label.

Reply:
did we get to conclusion here?

Reply:
@Ryun1 @Crypto2099 - I guess we've concluded by default to keep this suggestion in mind for our own submissions & readily recommend it to authors, to clarify these lists in future & updated CIP headers.  Does that sound fair enough?

I have a CIP-0001 update pending with issues over the last few months and will include this suggestion for official consideration in that PR, so we can officially define & settle it the syntax & associated recommendations there.